test/Dapr.E2E.Test/DaprTestApp.cs

```diff
@@ -19,8 +21,8 @@ public class DaprTestApp
         static string daprBinaryName = "dapr";
         private string appId;
         private bool useAppPort;
-        const string outputToMatchOnStart = "dapr initialized. Status: Running.";
-        const string outputToMatchOnStop = "app stopped successfully";
+        private readonly string[] outputToMatchOnStart = new string[]{ "You're up and running", };
+        private readonly string[] outputToMatchOnStop = new string[]{ "app stopped successfully", "failed to stop app id", };
```

Comment:
Need to handle the case where the app didn't properly start.

The existing code was leaking the process if something went wrong during startup. Ideally we'd also keep track of the PID and kill it if its still running when the tests finish.